### PR TITLE
AC: Fix several spelling errors

### PR DESCRIPTION
**Pull request by IRDonch**
A few of the things with errors in them weren't needed at all, so I just removed them.

One of these changes should actually reenable a test that has been accidentally disabled this whole time.

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md [hunk 1]
```diff
@@ -87,5 +87,4 @@ launchers:
       batch: 4
     mo_flags:
       - reverse_input_channels
-    cpu_extensions: cpu_extentions_avx512.so
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md:90]: we not remove functional opportunity to set extension, so I do not think that we should remove it from docs sample (as I know, it is still used for example for running on arm cpu)

**Review by eaidova — APPROVED** [review]

File: tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md (hunk 1 above)
Comment (IRDonch) [reply tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md:90]: I suppose I could keep the key, but what do I use for the value? The `cpu_extensions_avx512` library doesn't exist anymore.

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md:90]: cpu_extention.so or custom_cpu_extension.so will be ok (it is just abstract name used in example purposes, for running with extension, of course, user should compile them on own side)

Comment (IRDonch) [reply tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md:90]: Alright, updated.

**Comment by eaidova** [comment]
Jenkins please retry a build

**Pull request merged**